docs/user-guide/datastores.md

```diff
@@ -1,5 +1,56 @@
-** Consuming multiple data stores **
+---
+title: Datastore & backend layout
+---
 
-The service by default creates a volume in the datastore that hosts the VM. Optionally Docker volumes can be provisioned on a specific datastore by directly addressing the datastore when specifying the name in the format &lt;volume name>@&lt;datastore name>
+## Datastore & backend layout
 
-<script type="text/javascript" src="https://asciinema.org/a/80428.js" id="asciicast-80428" async></script>
+Docker volumes in vSphere are backed by VMDK files. These VMDKs could be backed by any storage backend. For example the backend storage could be one of NFS, SAN, VSAN, vVol. The VMDKs reside on a central location and can be reqested by any container running on any host. Ability to attach volumes from anywhere in cluster coupled with a clustering technology such as Swarm enables a highly available architecture.
+
+## Multiple datastores Support
+
+By default the volume is created in the default datastore for the VM. But there are use cases where you want to create and consume volumes from different datastores for specific performance or functionality. The vDVS allows you to create a volume on any of the datastores available.
+
+## Default datastore
+By default when you just specify name of volume - it is created on default datastore. 
+
```

Comment:
@lipingxue It will be great if you could review and give a better descriptions for default and custom datastores. Thanks!